### ISSUE TITLE
Unhandled rejection TypeError: Object.keys called on non-object

### ISSUE THREAD

**Issue by listepo** [tjunnone/npm-check-updates#116]
OS: Darwin MacBook-Air-Ivan.local 14.4.0 Darwin Kernel Version 14.4.0: Thu May 28 11:35:04 PDT 2015; root:xnu-2782.30.5~1/RELEASE_X86_64 x86_64
node: v0.12.7
npm: 2.13.3
run: ncu

```javascript
Unhandled rejection TypeError: Object.keys called on non-object
    at Function.keys (native)
    at Object.numParts (/Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/lib/version-util.js:24:49)
    at /Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/lib/versionmanager.js:78:32
    at /Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:7113:30
    at /Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2275:27
    at /Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3049:15
    at baseMap (/Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2274:7)
    at Function.sortBy (/Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:7112:20)
    at baseWrapperValue (/Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2768:30)
    at LodashWrapper.wrapperValue (/Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:6124:14)
    at upgradeDependencyDeclaration (/Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/lib/versionmanager.js:80:10)
    at wrapper (/Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3533:19)
    at /Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:8201:21
    at /Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3395:24
    at /Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3073:15
    at baseForOwn (/Users/ivan/.nvm/versions/node/v0.12.7/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2046:14)
```

**Comment by metaraine** [comment]
Thank you for reporting! I will investigate. Can you also post your package.json?

**Comment by listepo** [comment]
```json
{
  "name": "web",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git@bitbucket.org:log_money/web.git"
  },
  "author": "Ivan Tugay <listepo@gmail.com>",
  "license": "ISC",
  "devDependencies": {
    "del": "^1.2.0",
    "eslint": "^0.24.1",
    "eslint-config-standard": "^4.0.0-1",
    "gulp": "^3.9.0",
    "gulp-angular-templatecache": "^1.7.0",
    "gulp-concat": "^2.6.0",
    "gulp-eslint": "^0.15.0",
    "gulp-if": "^1.2.5",
    "gulp-sourcemaps": "^1.5.2",
    "gulp-uglify": "^1.2.0",
    "gulp-webserver": "^0.9.1"
  }
}
```
after npm cache clean and reinstall cannot replicate

**Comment by metaraine** [comment]
That’s okay. I should have a safeguard in place so that Object.keys can
never be called on a non-object anyway.

**Comment by josephfinlayson** [comment]
Getting this as well, can replicate after cache clean and reinstall

**Comment by metaraine** [comment]
@josephfinlayson Can you provide the package.json with which you can replicate this?

**Comment by metaraine** [comment]
Might have been fixed in `v2.0.02`, but not sure until replicable package.json is provided.

**Comment by opportunato** [comment]
Had this issue recently, upgraded to v2.0.4, all problems solved.
